src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let sum;
+
+  if (days >= 1 && days < 3) {
```

Comment:
Since `days` is a parameter, it's guaranteed to be at least `1`. So, the condition `days >= 1` is redundant and can be removed.